BUILD.bazel

```diff
@@ -102,20 +108,16 @@ cc_library(
         ],
         "//conditions:default": [],
     }),
-    features = select({
-        ":windows": ["windows_export_all_symbols"],
-        "//conditions:default": [],
-    })
 )
 
 cc_library(
     name = "gtest_main",
     srcs = ["googlemock/src/gmock_main.cc"],
-    deps = [":gtest"],
     features = select({
         ":windows": ["windows_export_all_symbols"],
         "//conditions:default": [],
-    })
+    }),
+    deps = [":gtest"],
```

Comment:
Looks like there is no good reason to move this line from previous location.Could you please limit the changes to the actual factual changes to remove the noise.